docs/architecture/OBSERVABILITY_ARCHITECTURE.md

```
@@ -0,0 +1,747 @@
+    name: string;
+    message: string;
+    category?: PDErrorCategory;  // 来自 ERROR_ARCHITECTURE
+    stack?: string;
+    cause?: PDLogEntry['error'];
+  };
+}
+```
+
+### 2.3 component 命名规范
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)